### ISSUE TITLE
🐛 Bug: Existing description and badges not detected

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/create-typescript-app#751]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

When migrating an existing repository, if the README.md already contains centered paragraphs for description and/or badges, they shouldn't be replaced.

### Actual

I tried migrating a package that already had them and they were duplicated:

```diff
<h1 align="center">All Contributors Auto Action</h1>

<p align="center">Fills in missing allcontributors entries for a repository. 👪</p>

+ <p align="center">
+ 	<a href="#contributors" target="_blank">
+ <!-- prettier-ignore-start -->
+ <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+ <img alt="All Contributors: 1" src="https://img.shields.io/badge/all_contributors-1-21bb42.svg" />
+ <!-- ALL-CONTRIBUTORS-BADGE:END -->
+ <!-- prettier-ignore-end -->
+ </a><a href="https://codecov.io/gh/JoshuaKGoldberg/all-contributors-auto-action" target="_blank">
+ 	<img alt="Codecov Test Coverage" src="https://codecov.io/gh/JoshuaKGoldberg/all-contributors-auto-action/branch/main/graph/badge.svg"/>
+ </a><a href="https://github.com/JoshuaKGoldberg/all-contributors-auto-action/blob/main/.github/CODE_OF_CONDUCT.md" target="_blank">
+ 	<img alt="Contributor Covenant" src="https://img.shields.io/badge/code_of_conduct-enforced-21bb42" />
+ </a><a href="https://github.com/JoshuaKGoldberg/all-contributors-auto-action/blob/main/LICENSE.md" target="_blank">
+ 	<img alt="License: MIT" src="https://img.shields.io/github/license/JoshuaKGoldberg/all-contributors-auto-action?color=21bb42">
+ </a><img alt="Style: Prettier" src="https://img.shields.io/badge/style-prettier-21bb42.svg" /><img alt="TypeScript: Strict" src="https://img.shields.io/badge/typescript-strict-21bb42.svg" />
+ </p>
+ 
+ <p align="center">Fills in missing allcontributors entries for a repository. 👪</p>
+ 
<p align="center">
	<a href="#contributors" target="_blank">
<!-- prettier-ignore-start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
<img alt="All Contributors: 1" src="https://img.shields.io/badge/all_contributors-1-21bb42.svg" />
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- prettier-ignore-end -->
	</a>
	<a href="https://codecov.io/gh/JoshuaKGoldberg/all-contributors-auto-action" target="_blank">
		<img alt="Codecov Test Coverage" src="https://codecov.io/gh/JoshuaKGoldberg/all-contributors-auto-action/branch/main/graph/badge.svg?token=eVIFY4MhfQ"/>
	</a>
	<a href="https://github.com/JoshuaKGoldberg/all-contributors-auto-action/blob/main/.github/CODE_OF_CONDUCT.md" target="_blank">
		<img alt="Contributor Covenant" src="https://img.shields.io/badge/code_of_conduct-enforced-21bb42" />
	</a>
	<a href="https://github.com/JoshuaKGoldberg/all-contributors-auto-action/blob/main/LICENSE.md" target="_blank">
	    <img alt="License: MIT" src="https://img.shields.io/github/license/JoshuaKGoldberg/all-contributors-auto-action?color=21bb42">
    </a>
	<a href="https://github.com/sponsors/JoshuaKGoldberg" target="_blank">
    	<img alt="Sponsor: On GitHub" src="https://img.shields.io/badge/sponsor-on_github-21bb42.svg" />
    </a>
	<img alt="Style: Prettier" src="https://img.shields.io/badge/style-prettier-21bb42.svg" />
    <img alt="TypeScript: Strict" src="https://img.shields.io/badge/typescript-strict-21bb42.svg" />
</p>
```

### Additional Info

Discovered on https://github.com/JoshuaKGoldberg/all-contributors-auto-action/tree/1ce7a4d2df5fd69765b7d407654847fde9e30340:

```shell
git clone https://github.com/JoshuaKGoldberg/all-contributors-auto-action
cd all-contributors-auto-action
git checkout 1ce7a4d2df5fd69765b7d407654847fde9e30340
pnpm i
npx create-typescript-app --base everything --mode migrate
```

It _might_ be the case that this only happens if there's a change to `--description` or some other data.
